teps/0089-nonfalsifiable-provenance-support.md

```diff
@@ -114,8 +133,8 @@ Roughly, this will look something like this:
 
 Each time the TaskRun is modified,
 * Controller verifies the TaskRun hasn't been modified
-* Controller requests a new SPIRE SVID and signature over the new modified TaskRun
-* Controller stores the new SVID and signature on the TaskRun as an annotation
+* Controller requests a new SPIRE SVID signs the new modified TaskRun
```

Comment:
typo i think? maybe something like "and then signs" or "and then uses it to sign"

Reply:
Yup - fixed!